test/core/fork/LockupLinear.t.sol

```diff
@@ -205,10 +231,12 @@ abstract contract Lockup_Linear_Fork_Test is Fork_Test {
         assertEq(lockup.getSender(vars.streamId), params.sender, "sender");
         assertEq(lockup.getStartTime(vars.streamId), params.timestamps.start, "startTime");
         assertFalse(lockup.wasCanceled(vars.streamId), "wasCanceled");
+        assertEq(lockup.getUnlockAmounts(vars.streamId).start, params.unlockAmounts.start, "unlockAmounts.start");
+        assertEq(lockup.getUnlockAmounts(vars.streamId).cliff, params.unlockAmounts.cliff, "unlockAmounts.cliff");
 
         // Assert that the stream's status is correct.
         vars.actualStatus = lockup.statusOf(vars.streamId);
-        if (params.timestamps.end <= vars.blockTimestamp) {
+        if (vars.streamedAmount == vars.createAmounts.deposit) {
```

Comment:
because the status is calculated based on an amount and not on time (expect if it is `PENDING`):

https://github.com/sablier-labs/v2-core/blob/6a8ec1318f598c351606eb196b9ccebed7dedbab/src/core/abstracts/SablierLockupBase.sol#L513-L516

similar to dynamic, where you can have 3 segments stream with: `s[0].amount + s[1].amount == depositedAmount && s[2].amount == 0`, here is the same, you can have the `unlockAmounts.start + unlockAmounts.cliff == deposited`, so it does not matter the `block.timestamp`

Reply:
Fair point.